### PR TITLE
Adding Singly Linked List in Java

### PR DESCRIPTION
**Pull request by Khushi2402**

### CODE REVIEW

File: Java/Queue.java [hunk 1]
```diff
@@ -16,6 +16,7 @@ boolean isEmpty()
         if (front == -1)
         {
             return true;
+
```
Comment (ShehrozIrfan) [inline Java/Queue.java]: Need to remove this extra line.

**Review by ShehrozIrfan — COMMENTED** [review]
Overall looks good.
- Remove the extra line
- Also need to resolve the conflicts

File: Java/Queue.java (hunk 1 above)
Comment (Khushi2402) [reply Java/Queue.java]: Okay I will remove it

**Comment by Khushi2402** [comment]
Done

**Pull request closed without merging**